.github/workflows/prod.yml

```diff
@@ -21,7 +21,7 @@ jobs:
     steps:
       - uses: actions/checkout@v4
         with:
-          fetch-depth: 2
+          fetch-depth: 25
```

Comment:
Hvor mange commits vi skal hente. 2 stammer fra dev-deploy, der vi sammenligner siste commit med nest siste commit. Her henter vi de 25 siste commitsene og satser på at de inneholder den nest siste tagen. Den siste tagen er den vi nettopp lagde for å lage release.